web-ui/.prettierignore

```diff
@@ -0,0 +1 @@
+_variables.css
```

Comment:
What does Prettier do to this that is bad?

Reply:
It breaks up the color-mix variables, not dire but I made a preference call.